helm/cert-manager-app/charts/cert-manager-giantswarm-clusterissuer/templates/cm.yaml

```diff
@@ -1,3 +1,40 @@
+{{- define "clusterissuer" }}
```

Comment:
I moved this into template because I need to render proper solver depending on configuration